Tests/FunctionalTests/PluginTests.swift

```diff
@@ -299,13 +299,18 @@ final class PluginTests {
 
     @Test(
         .enabled(if: (try? UserToolchain.default)!.supportsSwiftConcurrency(), "skipping because test environment doesn't support concurrency"),
-        .enabled(if: ProcessInfo.hostOperatingSystem == .macOS, "Test is only supported on macOS")
+        .enabled(if: ProcessInfo.hostOperatingSystem == .macOS, "Test is only supported on macOS"),
+        arguments: [BuildSystemProvider.Kind.native, .swiftbuild]
```

Comment:
suggestion (non-blocking): Consider augmenting the test to run against all build configuration.

```suggestion
        arguments: [BuildSystemProvider.Kind.native, .swiftbuild], BuildConfiguraiton.allCases
```